packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/workbench-views/details/DetailsView.types.ts

```diff
@@ -19,6 +20,6 @@ export interface DetailsViewState {
   pinned: boolean;
 }
 
-export interface DetailsViewConfiguration {
+export interface DetailsViewConfiguration extends WorkbenchViewConfiguration {
```

Comment:
`ExplorerViewConfiguration` is already extending `WorkbenchViewConfiguration`, I assume it should also be the case for `DetailsViewConfiguration`